sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/operations/_operations.py

```diff
@@ -30,7 +31,74 @@
     T = TypeVar("T")
     ClsType = Optional[Callable[[PipelineResponse[HttpRequest, HttpResponse], T, Dict[str, Any]], Any]]
 
-
+_SERIALIZER = Serializer()
+# fmt: off
+
+def build_query_knowledge_base_request(
+    **kwargs  # type: Any
+):
+    # type: (...) -> HttpRequest
+    content_type = kwargs.pop('content_type', None)  # type: Optional[str]
+    project_name = kwargs.pop('project_name')  # type: str
+    deployment_name = kwargs.pop('deployment_name', None)  # type: Optional[str]
+
+    api_version = "2021-07-15-preview"
+    accept = "application/json"
+    # Construct URL
+    url = kwargs.pop("template_url", '/:query-knowledgebases')
```

Comment:
FYI: this might have to change. There's a swagger board review coming up soon as guidelines have changed to NOT have a forward slash. I'm hoping either is supported, though.